MidCamp/midcamp
# Adding config split for local, dev modules.

### Changes

* Adds the config_split module
* Creates a local config split with devel, kint, and field/views UI modules
* Remove the tour module
* Disable local split on production, disabling kint, devel, dblog, and UI modules
* Add patch to prevent sensitive, ignored config from being exported

### Testing steps

* Install site and pull down database from production.
* Run `drush cim` from within the container.
    * This should enable kint and devel during testing since other modules in the split are already enabled on prod
* Run `drush cex` and notice that mailchimp and recaptcha config is not changed to include sensitive keys
* Change the environment in `docker-compose.yml` to `production` by uncommenting line `19 #LAGOON_ENVIRONMENT_TYPE: production` and run `docker-compose up -d` to rebuild containers with new environment variables
* Clear caches and run `drush cim`.  dblog, devel, kint, and others should be disabled based on being in the production environment.
